### PR TITLE
WH lint fix

### PR DESCRIPTION
**Pull request by arnabp92**
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-transformer/170)

### CODE REVIEW

File: v0/destinations/util.js [hunk 1]
```diff
@@ -66,6 +53,7 @@ const updatePayload = (currentKey, eventMappingArr, value, payload) => {
     if (obj.rudderKey === currentKey) {
```
Comment (SumanthPuram) [inline v0/destinations/util.js:53]: Instead of map, we should be using a while/for-each construct here.

File: versionedRouter.js [hunk 2]
```diff
@@ -114,7 +118,7 @@ if (startDestTransformer) {
 
       const transformedEvents = [];
       await Promise.all(
-        Object.entries(groupedEvents).map(async ([dest, destEvents]) => {
+        Object.entries(groupedEvents).map(async ([destEvents]) => {
```
Comment (SumanthPuram) [inline versionedRouter.js:121]: This needs to be fixed. The first argument will be destID and the second is list of events.

Example here: https://repl.it/repls/SmoggyNextCrash

File: warehouse/v0/util.js [hunk 3]
```diff
@@ -10,9 +10,10 @@ const toSnakeCase = str => {
 };
 
 const toSafeDBString = str => {
-  if (parseInt(str[0]) > 0) str = `_${str}`;
-  str = str.replace(/[^a-zA-Z0-9_]+/g, "");
-  return str.substr(0, 127);
+  let parsedStr = str;
+  if (parseInt(str[0], 10) > 0) parsedStr = `_${str}`;
```
Comment (SumanthPuram) [inline warehouse/v0/util.js:14]: Make it more readable?
```
if (parseInt(str[0], 10) > 0) {
  parsedStr = `_${str}`;
}
 
```

File: v0/destinations/util.js (hunk 1 above)
Comment (arnabp92) [reply v0/destinations/util.js:53]: Done

File: versionedRouter.js (hunk 2 above)
Comment (arnabp92) [reply versionedRouter.js:121]: Done

File: warehouse/v0/util.js (hunk 3 above)
Comment (arnabp92) [reply warehouse/v0/util.js:14]: Done

**Pull request merged**